tonic/models/vic/vic.py

```diff
@@ -67,8 +67,18 @@ def run(self, global_param, logdir=None):
 
         return self.returncode
 
-    def _call_vic(self, *args):
-        vic_args = [self.executable] + [a for a in args]
+    def _call_vic(self, *args, mproc_num=None):
+
+	vic_args = []
+
+	if mproc_num is not None:
+		if  type(mproc_num) != int:  
+			raise AssertionError 
+			print("number of processors has not been specified as an integer") 
```

Comment:
this is not how you raise an error in Python. You will never see this print statement.  It should also be a `TypeError`:

```Python
if isinstance(mpi_proc, int):
    raise TypeError("number of processors must be specified as an integer")
```